Makefile

```diff
@@ -0,0 +1,22 @@
+.PHONY: fmt
+fmt:
+	uv run ruff format src
+
+.PHONY: check-fmt
+check-fmt:
+	uv run ruff format --check --diff
+
+.PHONY: ruff
+ruff:
+	uv run ruff check src
+
+.PHONY: mypy
+mypy:
+	uv run mypy src/acron
+
+.PHONY: lint
+lint: mypy ruff check-fmt
+
+.PHONY: build
```

Comment:
This shouldn't be phony. It should point to the real build/ directory that gets created. And possibly a make clean target could be nice as well.